dev/Repeater/InspectingDataSource.cpp

```diff
@@ -22,31 +22,41 @@ InspectingDataSource::InspectingDataSource(const winrt::IInspectable& source)
     }
     else
     {
-        // The bindable interop interface are abi compatible with the corresponding
-        // WinRT interfaces.
-        auto bindableVector = source.try_as<winrt::IBindableVector>();
-        if (bindableVector)
+        const auto vectorView = source.try_as<winrt::IVectorView<winrt::IInspectable>>();
+        if (vectorView)
```

Comment:
curious about the choice of order. Should we check for this after IBindableVector but before IIterable<insp> ? I think it makes sense to do this check before iterable because for iterable, we have to enumerate. @mikehill in case he has any thoughts about the order of the checks.

Reply:
I've moved it behind the IBindableVector now, though I think it will rarely make a difference.